mautic/mautic
# [UI] Minimalist tables for main pages

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Improvement based on @npracht's suggestion on [Slack ](https://mautic.slack.com/archives/C024XBCQS3Z/p1704603197115639)

| Before                                      | After
| -------------------------------------- | ---
| ![image](https://github.com/mautic/mautic/assets/116097999/3002c6d1-7fa2-4df2-82f9-cf1d28978d1b)           | ![image](https://github.com/mautic/mautic/assets/116097999/a3377ef5-a568-457b-b51a-2406c43babb9)

It also has a micro transition for bg color

https://github.com/mautic/mautic/assets/116097999/78d03a55-4bf8-4765-9e54-4693903f7ad6




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Stages > Create 2 > Look at the list
